PiwikTracker/PiwikTracker.swift

```diff
@@ -55,6 +57,7 @@ final public class PiwikTracker: NSObject {
     
     internal func queue(event: Event) {
         guard !isOptedOut else { return }
+        logger?.verbose(){ return "Queued event: \(event)" }
```

Comment:
Oh! I didn't know about the `@autoclosure`. It's perfect for this use case. Will add it.